tests/bsim/bluetooth/host/adv/periodic/src/per_adv_syncer.c

```diff
@@ -253,6 +253,33 @@ static void main_per_adv_syncer(void)
 	PASS("Periodic advertising syncer passed\n");
 }
 
+static void main_per_adv_syncer_app_not_scanning(void)
+{
+	int err;
+	struct bt_le_per_adv_sync *sync = NULL;
+
+	common_init();
+	start_scan();
+
+	printk("Waiting for periodic advertising...\n");
+	WAIT_FOR_FLAG(flag_per_adv);
+	printk("Found periodic advertising.\n");
+
+	printk("Stopping scan\n");
+	err = bt_le_scan_stop();
+	if (err != 0) {
+		FAIL("Failed to stop scan: %d", err);
+		return;
+	}
+
+	create_pa_sync(&sync);
+
+	printk("Waiting for periodic sync lost...\n");
+	WAIT_FOR_FLAG(flag_per_adv_sync_lost);
+
+	PASS("Periodic advertising syncer passed\n");
```

Comment:
can you `PASS_AND_EXIT()` instead? That will early-return the simulation, saving CI cycles